scripts/openstack-quickstart-demosetup

```diff
@@ -506,6 +506,20 @@ if [ "x$with_tempest" = "xyes" ]; then
     crudini --set /etc/neutron/neutron_lbaas.conf service_providers service_provider "LOADBALANCERV2:Haproxy:neutron_lbaas.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default"
 fi
 
+# NOTE(toabctl): epmd only listens on the loopback interface which does not work
```

Comment:
the default systemd socket activation only listens on the loopback interface.. which causes rabbitmq to try to start its own epmd